src/api/GraphSchema/types.ts

```diff
@@ -129,6 +130,7 @@ export interface GraphSchemaProperty {
   required: boolean;
   visibility: DataVisibility;
   propertyType: PropertyType;
+  indexed: boolean | null;
```

Comment:
@mzorkany don't use null, indexed is always defined for the frontend.

and if it wasn't in some cases, you should use indexed?: boolean with a comment explainin why.